TData/Compile/Comp10.pas

```diff
@@ -4,12 +4,12 @@
   
 function D2R (deg: real): real;
 begin
-   D2R := deg * PI / 180;
+   D2R := deg * Pi / 180;

```

Comment:
No - the code should work even with `PI`, or `pi` : Pascal is case -insensitive

Reply:
This uppercase PI in .pas and the library has Pi. So IL code looks as below. But I still get _Field not found: 'PSILib.Lib.Pi'_
.class Program {
    .field static int32 n
  .method static float64 D2R (float64 deg) {
    ldarg deg
    **ldsfld float64 [PSILib]PSILib.Lib::Pi**
    mul
    ldc.i4 180
    conv.r8
    div
    ret
  }

Reply:
If you see PSILib, you will see that all the 'variables' there are implemented as properties (that's because some of them like window-width etc could change over time). You need to therefore do a 'property read'. For example:
`call float64 [PSILib]PSILib.Lib::get_PI ()`

Reply:
Done